### PR TITLE
Improve mobile page speed

### PR DESCRIPTION
**Pull request by cp-dharti-r**
## Review checklist

**Note:** Make sure all points should be checked before submiting PR for review, otherwise PR will be cosidered invalid.

- [x] Follow guidelines for creating branch name, adding commit messages or PR title
- [x] Add MR description if applicable
- [x] Follow guidelines for naming conventions everywhere (i.e files/folders, data structures)
- [x] Reuse code (if the same code is written twice, make it common and reuse it at both places)
- [x] Remove unused or commented code if not required
- [x] Make MR self-approved - review MR as a reviewer and give it self-approval if everything is ok, if not make the required changes
- [x] In progress MR should be marked as draft
- [x] Make MR mark as ready before submitting it for review
- [x] Add images, videos, or gifs of UI
- [x] Add page speed score screenshot of UI (It should be > 85 for mobile and > 90 for desktops)

**Design should be tested on:**

- [ ] Inspector (all iPhones and android devices like samsung, motorola)
- [ ] Desktop (chrome, firefox, safari)
- [ ] Laptop / Macbook (chrome, firefox, safari)
- [ ] iPhone(safari)
- [ ] Available android devices (chrome, firefox)
- [ ] iPad(safari)
- [ ] Tablet (chrome, firefox)

## PR Description
- Improve mobile page speed (deployed in dev to check)
https://pagespeed.web.dev/analysis/https-dev-stack-canopas-com/c9bpjdoj9v?form_factor=mobile
- Added home page required images into public folder to get show images faster and also easy to get from any files
- Some UI fixes in portfolio and portfolio detail page

### CODE REVIEW

File: nuxt-frontend/pages/portfolio/[id].vue [hunk 1]
```diff
@@ -133,3 +133,4 @@ watch(
   }
 }
 </style>
+~/json-data/luxeradio-data.js~/json-data/justly-data.js~/json-data/togness-data.js
```
Comment (cp-sumi-k) [inline nuxt-frontend/pages/portfolio/[id].vue:136]: Can you explain this?

File: nuxt-frontend/package.json [hunk 2]
```diff
@@ -43,7 +43,6 @@
   },
   "dependencies": {
     "@aws-sdk/client-s3": "^3.388.0",
-    "@nuxtjs/robots": "^3.0.0",
```
Comment (cp-sumi-k) [inline nuxt-frontend/package.json:46]: Please update branch with master

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

**Comment by cp-sumi-k** [comment]
There were many images added, Please add reason for adding images in description

File: nuxt-frontend/pages/portfolio/[id].vue (hunk 1 above)
Comment (cp-dharti-r) [reply nuxt-frontend/pages/portfolio/[id].vue:136]: Maybe it's mistakenly added while squashing. Removed it

**Review by cp-sumi-k — DISMISSED** [review]

**Review by cp-sumi-k — APPROVED** [review]

**Review by nidhi-canopas — APPROVED** [review]

**Pull request merged**